pyproject.toml

```diff
@@ -19,10 +19,15 @@ scipy = [
 ]
 pyarrow = "^16.0.0"
 pyyaml = "^6.0.1"
+fire = "^0.6.0"
+jinja2 = "^3.1.4"
 
 [tool.poetry.group.dev.dependencies]
 pytest = "^8.2.0"
 
+[tool.poetry.scripts]
+cosmicqc = "cosmicqc.cli:cli_analyze"
```

Comment:
What is this section for?

Reply:
Thanks for this comment - this helps associate `cosmicqc` as a CLI tool through `cosmicqc.cli.cli_analyze` via poetry's installation.